TF_Application/Assets/TouchFree/Tooling/Scripts/Connection/ConnectionManager.cs

```diff
@@ -55,6 +55,8 @@ public static ServiceConnection serviceConnection
         // This value is settable in the Inspector.
         [SerializeField] string port = "9739";
 
+        static bool disconnected = true;
```

Comment:
should this be 'connected' rather than 'disconnected'?

Reply:
I mean more to remove the idea of _wanting_ it to be false, I would typically prefer to know if I was connected and then fix it if I am _not_ connected, rather than looking for _not_ being disconnected